src/Main.java

```diff
@@ -9,7 +9,7 @@ public class Main {
     static HashMap<String, ArrayList<Integer>> mapOfWords;
     static Scanner sc = new Scanner(System.in);
 
```

Comment:
The Main class should do nothing except call other methods to start the program(you may have a programController class for this purpose).

Reply:
That's right... Done!